configure.ac

```diff
@@ -1108,7 +1108,10 @@ AC_SUBST([private_namespace])
 dnl Do not add suffix to installed files by default.
 AC_ARG_WITH([install_suffix],
   [AS_HELP_STRING([--with-install-suffix=<suffix>], [Suffix to append to all installed files])],
-  [INSTALL_SUFFIX="$with_install_suffix"],
+  [case "$with_install_suffix" in
+   *\ * ) AC_MSG_ERROR([Install suffix should not contain spaces]) ;;
```

Comment:
Just saw that the answer is in the summary:

> This doesn't address spaces in prefix though, but make install doesn't work in this case either

Reply:
I can do that to prefix, I was a little hesitant though. But really, if it doesn't work, why allowing that too. If there are any problems with restricting the prefix like that, we can easily revert such change.

Reply:
Thanks for adding it for the prefix. I'm fine with adding or not either way -- up to you there. I was asking because I forgot spaces wouldn't work anyway.